### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by OndrejSpanel**
Trying how Docker would work ....

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -4,4 +4,6 @@ addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.33")
 
 addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.0.0")
 
-addSbtPlugin("com.codecommit" % "sbt-github-packages" % "0.5.3")
\ No newline at end of file
+addSbtPlugin("com.codecommit" % "sbt-github-packages" % "0.5.3")
+
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.9.11")
```
Comment (guizmaii) [inline project/plugins.sbt:9]: It's `addSbtPlugin("com.github.sbt" % "sbt-native-packager" % "1.9.11")`

The namespace is incorrect. Can you push a PR on the `sbt-native-packager` to fix the doc, please?

Comment (OndrejSpanel) [reply project/plugins.sbt]: > Can you push a PR on the `sbt-native-packager` repo to fix the doc, please?

I doubt I am able to do that, as I have no idea where the documentation sources are located.

Comment (guizmaii) [reply project/plugins.sbt]: Hum. Apparently it's fixed in the latest version of the doc: https://sbt-native-packager.readthedocs.io/en/latest/index.html

Comment (OndrejSpanel) [reply project/plugins.sbt]: Unfortunately the one I have coped the code from (https://www.scala-sbt.org/sbt-native-packager/) is higher ranked in Google results

Comment (guizmaii) [reply project/plugins.sbt]: FYI https://users.scala-lang.org/t/https-www-scala-sbt-org-sbt-native-packager-is-oudated/8902

Comment (OndrejSpanel) [reply project/plugins.sbt]: Thanks.

**Pull request merged**